cmd/network/verification.go

```diff
@@ -195,32 +201,32 @@ func (e *EgressVerification) setup(ctx context.Context) (*aws.Config, error) {
 // Its input is an OCM internal/external ClusterId and it returns the corresponding input to osd-network-verifier with
 // default AWS tags, one of the cluster's private subnet IDs, and the cluster's master security group.
 // Can override SecurityGroupId and SubnetId.
-func (e *EgressVerification) generateAWSValidateEgressInput(ctx context.Context, region string) (*onv.ValidateEgressInput, error) {
+func (e *EgressVerification) generateAWSValidateEgressInput(ctx context.Context, region string) (*onv.ValidateEgressInput, []*onv.ValidateEgressInput, error) {
```

Comment:
```suggestion
func (e *EgressVerification) generateAWSValidateEgressInput(ctx context.Context, region string) ([]*onv.ValidateEgressInput, error) {
```

It changes code in this function when returning, but lets only return the slice of `ValidateEgressInput` since we won't need the first return value anymore